### DISCUSSION TITLE
How to Create a Custom 404 Error Page in Nextjs 13

### DISCUSSION THREAD

**Discussion by giscus** [Evavic44/victoreke.com#69] [Announcements]
# How to Create a Custom 404 Error Page in Nextjs 13

Learn how to create a custom 404 Error page in Next.js 13 app directory to handle all unmatched URLs and for nested routes using the not-found.js file

https://victoreke.com/blog/how-to-create-a-custom-404-error-page-in-nextjs-13

**Comment by Evavic44** [comment]


**Reply by Evavic44** [reply]
```js
console.log("It works")
```

**Comment by RafaelN0bre** [comment]
Hey @Evavic44, could you share the css's for the page?

**Reply by Evavic44** [reply]
Here it is. [GitHub Repository](https://github.com/Evavic44/nextjs-13-custom-404/blob/main/app/styles/globals.css)

**Comment by NourAlzway** [comment]
Nice Article :) !
